schemas/sampler_result_v2_schema.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "http://www.qiskit.org/schemas/sampler_result_v2_schema.json",
+    "title": "EstimatorV2 result",
+    "description": "The result for an SamplerV2 API call",
+    "version": "1.0.0",
+    "type": "object",
+    "required": ["results", "metadata"],
+    "properties": {
+        "results": {
+            "type": "array",
+            "description": "Array of results, one for each PUB in the original job",
+            "items": {
+                "type": "object",
+                "description": "A single PUB result, containing result data and metadata",
+                "required": ["data", "metadata"],
+                "properties": {
+                    "data": {
+                        "type": "object",
+                        "description": "PUB results for sampler are split according to the classical bits register names. For each register, an array of samples (in hexadecimal string format) is returned",
+                        "additionalProperties": {
+                            "type": "object",
+                            "description": "The data corresponding to a specific measurement",
+                            "properties": {
+                                "samples": {
+                                    "type": "array",
+                                    "description": "An array of samples (given as hexadecimal strings)",
+                                    "items": {"type":  "string"}
```

Comment:
When do we return a `samples` array? I thought sampled data is all grouped under register names?

Reply:
Yes, but currently the way `BitArray` is converted to JSON output is as an object with a `samples` field containing the flattening of the bit array and the number of bits.